lib/travis/api/serialize/serializer.rb

```diff
@@ -1,5 +1,20 @@
 require 'active_model_serializers'
 
+# This hideousness courtesy of http://stackoverflow.com/a/8339255
+module ActiveSupport::JSON::Encoding
+  class << self
+    def escape(string)
```

Comment:
Style wise I might suggest using def self.escape instead of class << self since this is a single method